QRServer/lobby/lobbyclient.py

```diff
@@ -51,20 +52,27 @@ def _handle_qrl(self, values):
     def _handle_join_lobby(self, values):
         username = values[1].decode('utf8')
         password = values[2].decode('utf8')
-        if not DBConnector().user_exists(username, password):  # FIXME implement dbconnector
-            self.send(b'<L>~<BAD_MEMBER>\x00')
-            self.close()
-        elif self.lobby_server.username_exists(username):
+
+        if not utils.is_guest(username):
+            user_id = connector().authenticate_member(username, password)
+            if user_id is None:
+                log.info('Player {} tried to connect, but failed to authenticate'.format(username))
```

Comment:
Is loglevel Info necessary for this? I'd say more of a debug

Reply:
sure, changed to `debug`